src/Numerics/DGMethods/DGModel.jl

```diff
@@ -87,10 +105,6 @@ function (dg::DGModel)(
     ndrange_interior_surface = Nfp * length(grid.interiorelems)
     ndrange_exterior_surface = Nfp * length(grid.exteriorelems)
 
-    if !increment && num_state_conservative < num_state_tendency
-        tendency .= -zero(FT)
-    end
-
```

Comment:
We might still need something like this for when we mask out the tracers. @lcw?

Reply:
yup, I just realised what that was for.